packages/boot/src/types.ts

```diff
@@ -125,3 +126,33 @@ export interface Bootable {
    */
   booters(...booterClasses: Constructor<Booter>[]): Binding[];
 }
+
+/**
+ * `@booter` decorator to mark a class as a `Booter` and specify the artifact
+ * namespace for the configuration of the booter
+ *
+ * @example
+ * ```ts
+ * @booter('controllers')
+ * export class ControllerBooter extends BaseArtifactBooter {
+ *   constructor(
+ *     @inject(CoreBindings.APPLICATION_INSTANCE) public app: Application,
+ *     @inject(BootBindings.PROJECT_ROOT) projectRoot: string,
+ *     @config()
+ *    public controllerConfig: ArtifactOptions = {},
+ *   ) {
+ *   // ...
+ *   }
+ * }
+ * ```
+ *
+ * @param artifactNamespace - Namespace for the artifact. It will be used to
+ * inject configuration from boot options
+ * @param specs - Extra specs for the binding
+ */
+export function booter(artifactNamespace: string, ...specs: BindingSpec[]) {
```

Comment:
I don't really get why we name it `artifactNamespace` here. But I either don't know how we usually name these properties.. So I guess it's fine. 
Could you make the example above more specific, or add a comment for this change? It would help to understand the code/naming logic. Thanks!

Reply:
Improved the tsdocs for the decorator function.